docs/Makefile

```diff
@@ -15,6 +15,7 @@ help:
 
 clean:
 	rm -rfv "$(SOURCEDIR)/_autosummary"
+	rm -rfv "$(SOURCEDIR)/_examples_database"
```

Comment:
Maybe, because this is costly to re-generate, add a new `cleanall` target that performs `clean` + that line?